root/defaults/nginx/location-confs/proxy.conf.sample

```diff
@@ -25,7 +25,11 @@ proxy_set_header Host $host;
 proxy_set_header Proxy "";
 proxy_set_header Upgrade $http_upgrade;
 proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
-proxy_set_header X-Forwarded-Host $host;
-proxy_set_header X-Forwarded-Proto https;
+proxy_set_header X-Forwarded-Host $host:$server_port;
```

Comment:
https://www.nginx.com/resources/wiki/start/topics/examples/likeapache/#nginx
We should test to confirm this doesn't cause widespread issues.

Reply:
Tested, no issues.